tests/apps_test.go

```diff
@@ -101,30 +102,33 @@ var _ = Describe("Apps", func() {
 		var appName string
 
 		BeforeEach(func() {
+			os.Chdir("example-go")
 			appName = getRandAppName()
 			cmd := createApp(appName)
 			Eventually(cmd).Should(SatisfyAll(
 				Say("Git remote deis added"),
 				Say("remote available at ")))
-
+			Eventually(cmd).Should(Exit(0))
 			cmd, err := start("GIT_SSH=%s git push deis master", gitSSH)
```

Comment:
Worth setting `cmd.Dir = "example-go"` instead of doing the `Chdir`? If so, not blocking for this PR, just a comment.